styles/components/Search.module.scss

```diff
@@ -28,6 +28,12 @@
 .advancedSearch {
   align-self: flex-start;
   min-width: 127px;
+  line-height: 1;
+  margin-left: auto;
+
+  @media screen and (min-width: 600px) {
+    margin-top: -8px;
+  }
```

Comment:
The margin between the searchbar and this link should be 16px which is what the parent is doing. So this doesn't seem like it's needed.